packages/pythia-crypto/src/initPythia.ts

```diff
@@ -1,8 +1,15 @@
 import initPythiaModules from '@virgilsecurity/core-pythia';
 
-import { setPythiaModules } from './pythiaModules';
+import { getPythiaModules, setPythiaModules } from './pythiaModules';
 
 export async function initPythia() {
+  try {
+    getPythiaModules();
+    // eslint-disable-next-line no-console
+    console.warn('Pythia modules are already set. Further calls to `initPythia` are ignored.');
+    return;
+    // eslint-disable-next-line no-empty
+  } catch (_) {}
```

Comment:
Let's just add a helper function to tell us if modules are set. This code is confusing.